pkg/dump/dump.go

```diff
@@ -487,6 +487,21 @@ func getProxyConfiguration(ctx context.Context, group *errgroup.Group,
 		return nil
 	})
 
+	group.Go(func() error {
+		partials, err := GetAllPartials(ctx, client, config.SelectorTags)
+		if err != nil {
+			return fmt.Errorf("partials: %w", err)
+		}
+
+		partials, err = excludeKonnectManagedEntities(partials)
+		if err != nil {
+			return fmt.Errorf("partials: %w", err)
```

Comment:
Just to differentiate these 2 errors:

```suggestion
			return fmt.Errorf("failed excluding Konnect managed entities from partials: %w", err)
```